first/expo/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Farmington Valley FIRST Expo
+---
+
+<!--<h2>Farmington Valley FLL Jr.</h2>
+This section will be updated with more information about the Farmington Valley FLL Jr. programs soon.
+-->
+<h2>Farmington Valley <i>FIRST</i> Expo</h2>
```

Comment:
```suggestion
## Farmington Valley _FIRST_ Expo
```
Since you've changed the filetype to Markdown, it would make sense to update things that can be done in Markdown instead of HTML to Markdown.